packages/gdrcopy.spec

```diff
@@ -58,7 +58,7 @@ echo "building"
 make -j CUDA=${CUDA}
 
 %install
-make install DESTDIR=$RPM_BUILD_ROOT PREFIX=%{_prefix}
+make install DESTDIR=$RPM_BUILD_ROOT prefix=%{_prefix} libdir=/usr/lib64
```

Comment:
Would be good to set `libdir=${_libdir}` so that it is consistent with the config under `%files` section.

Reply:
Fixed, thanks!